keyboards/minidox/keymaps/dustypomerleau/keymap.c

```diff
@@ -166,7 +166,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
  *                                `------'    `------'
  */
 [_SYS] = LAYOUT( \
-  RESET,   DEBUG,   QWERTY,  CMK_DHM, _______,       _______,  KC__VOLDOWN, KC__VOLUP, _______,  _______, \
+  RESET,   DEBUG,   QWERTY,  CMK_DHM, _______,       _______,  KC__VOLDOWN, KC__VOLUP, KC_BRID,  KC_BRIU, \
```

Comment:
If you want these volume keys to work on other OSes, you can change them to `KC_VOLD`, `KC_VOLU` and `KC_MUTE`, and the `KC_POWER` key to `KC_PWR`.

Reply:
You are saying that `KC_VOLD` works on both macOS and Windows? If that’s the case, I’m not sure I understand the rationale for providing `KC__VOLDOWN`.

Reply:
Basically, just for completeness. There are a few keycodes that, for whatever reason, are not recognised on certain OSes. You can see in the keycode lists, especially [the media keys](https://docs.qmk.fm/#/keycodes_basic?id=media-keys) that they are marked with the supported OS in parentheses. Otherwise they can be assumed to work universally.

Reply:
I’m happy to make this change—but I won’t have time today to test and implement. Will push it tomorrow if there are no issues on my machine.

Reply:
It's only a suggestion, I was just pointing it out in case you didn't know :)

Reply:
I suspected this might be the case: The keycodes do work on macOS, but the macros I had included to make micro-volume adjustments only work with the mac-specific keycodes. So I have switched the standard volume keys with the universal ones, and left the smaller volume adjustments as macOS-specific.